ap/config/analysis_st.py

```diff
@@ -45,6 +45,99 @@
 # create a config by passing the campaign, so id and name will be identical
 config_2018 = analysis_st.add_config(campaign_2018)
 
+# location of JEC txt files
+config_2018.set_aux("jec", DotDict.wrap({
+    "source": "https://raw.githubusercontent.com/cms-jet/JECDatabase/master/textFiles",
+    "campaign": "Summer19UL18",
+    "version": "V5",
+    "jet_type": "AK4PFchs",
+    "levels": ("L1FastJet", "L2Relative", "L2L3Residual", "L3Absolute"),
+    "data_eras": ("RunA", "RunB", "RunC", "RunD"),
```

Comment:
Sure, if you like :) I'm a bit paranoid about having mutable things in configs just in case some piece of user code ends up changing the value without realizing, but that's perhaps unjustified here.